src/view/observer/mutationobserver.js

```diff
@@ -204,8 +204,8 @@ export default class MutationObserver extends Observer {
 		}
 
 		for ( const viewElement of mutatedElements ) {
-			const domElement = domConverter.mapViewToDom( viewElement );
 			const viewChildren = Array.from( viewElement.getChildren() );
+			const domElement = domConverter.mapViewToDom( viewElement );
```

Comment:
Why changing the order?

Reply:
At some point, I've mistakenly restored old code. I'll revert the original sequence.